numba/stencils/stencil.py

```diff
@@ -257,7 +257,10 @@ def add_indices_to_kernel(self, kernel, index_names, ndim,
                                                        const_index_vars[dim], loc)
                             new_body.append(ir.Assign(getitemcall, getitemvar, loc))
                             # Get the type of this particular part of the index tuple.
-                            one_index_typ = stmt_index_var_typ[dim]
+                            if isinstance(stmt_index_var_typ, types.ConstSized):
```

Comment:
Why is this change needed?

Reply:
nvm, I found the affected test `test_basic81 (numba.tests.test_stencils.TestManyStencils)`